src/viewer.c

```diff
@@ -578,8 +642,8 @@ static void draw_image(struct pixmap* wnd)
     // clear image background
     if (img->alpha) {
         if (ctx.image_bkg == GRID_BKGID) {
-            pixmap_grid(wnd, ctx.img_x, ctx.img_y, width, height,
-                        GRID_STEP, GRID_COLOR1, GRID_COLOR2);
+            pixmap_grid(wnd, ctx.img_x, ctx.img_y, width, height, GRID_STEP,
```

Comment:
This change is not needed.

Reply:
clang-format was moving this down. Fixed it manually